### PR TITLE
refactor: some small stuff

### PR DESCRIPTION
**Pull request by TheAlexLichter**

### CODE REVIEW

**Review by galvez — DISMISSED** [review]
Love this style :)

File: packages/webpack/src/builder.js [hunk 1]
```diff
@@ -226,9 +224,7 @@ export class WebpackBundler {
   }
 
   async unwatch() {
-    for (const watching of this.compilersWatching) {
-      await watching.close()
-    }
+    await Promise.all(this.compilersWatching.map(watching => watching.close()))
```
Comment (manniL) [inline packages/webpack/src/builder.js]: Not sure if order is important for unwatch?

Comment (pi0) [reply packages/webpack/src/builder.js]: no it isn't.

File: packages/webpack/src/utils/style-loader.js [hunk 2]
```diff
@@ -37,39 +37,50 @@ export default class StyleLoader {
...
   }
 
   css(options) {
     options.exportOnlyLocals = this.exportOnlyLocals
-    return [
-      ...options.exportOnlyLocals ? [] : [this.styleLoader()],
-      { loader: 'css-loader', options }
-    ]
+
+    const cssLoader = { loader: 'css-loader', options }
```
Comment (clarkdo) [inline packages/webpack/src/utils/style-loader.js]:
```suggestion
const cssLoader = { loader: 'css-loader', options }

if (options.exportOnlyLocals) {
  return [ cssLoader ]
}

return [ this.styleLoader(), cssLoader ]
```

Comment (manniL) [reply packages/webpack/src/utils/style-loader.js]: Neat idea!

**Review by husayt — DISMISSED** [review]
Really neat

File: packages/builder/src/builder.js [hunk 3]
```diff
@@ -61,7 +60,7 @@ export default class Builder {
       })
 
       // Close hook
-      this.nuxt.hook('close', () => this.close())
+      this.nuxt.hook('close', this.close)
```
Comment (clarkdo) [inline packages/builder/src/builder.js]: This may lose the context for `close`

Comment (manniL) [reply packages/builder/src/builder.js]: Damn, right! I wasn't sure about that change and remembered that we talked about that before.

**Review by atinux — DISMISSED** [review]
😍 
I love to see these refactors!

**Comment by manniL** [comment]
I hope CI passes. If that's the case, I'm done 🙌

**Pull request merged**